project-base/storefront/pages/_document.tsx

```
@@ -25,11 +173,50 @@ class MyDocument extends Document<MyDocumentInitialProps> {
 
     render() {
         const { htmlLang } = this.props;
+        const warningTranslation = getBrowserWarningTranslation(htmlLang);
 
         return (
             <Html lang={htmlLang}>
-                <Head />
+                <Head>
+                    <style dangerouslySetInnerHTML={{ __html: BROWSER_WARNING_STYLES }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)